cmd/runner/main.go

```diff
@@ -46,6 +46,12 @@ func main() {
 		log.Fatalf("Failed to validate concurrency levels: %v", err)
 	}
 
+	log.Printf("annotation key for queue assignment: %s", a)
+	log.Printf("polling interval: %v", p)
+	log.Printf("polling retries: %d", retries)
+	log.Printf("test counts per queue: %v", runner.CountConfigs(configQueueMap))
+	log.Printf("queue concurrency levels: %v", c)
```

Comment:
This is a question for me to learn, when should we capitalize the log message?  I thought only the error message should be start with lower case.

Reply:
I was trying to group them together visually, but uppercase is more consistent. Change it to uppercase.